static/app/views/automations/detail.tsx

```diff
@@ -79,9 +172,33 @@ function AutomationDetailContent({automation}: {automation: Automation}) {
         </DetailLayout.Header>
         <DetailLayout.Body>
           <DetailLayout.Main>
+            <TimeRangeSelector
```

Comment:
You can use `DatePageFilter` instead here and it will sync with the page filter store

Reply:
Oh but you may need to wrap the whole route in `PageFiltersContainer` to be able to do so